Sources/TuistKit/Generator/LinkGenerator.swift

```diff
@@ -266,3 +292,13 @@ final class LinkGenerator: LinkGenerating {
         pbxTarget.buildPhases.append(buildPhase)
     }
 }
+
+private extension XCBuildConfiguration {
```

Comment:
I'd move to `xcodeproj` *(the library)* and make it public. I think it's a method more developers could benefit from.

Reply:
👍 will submit a separate PR to xcodeproj for this, it would need to be beefed up a bit further as here we only deal with space separated string settings, however other use cases may contain array of strings.